.github/workflows/update_bundled_useragents.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Bundled User Agents
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # Run weekly on Sunday at midnight
```

Comment:
Will it create a new PR every week? THat would be bit too much for me to handle constantly?

Reply:
In principle no. In the action's [README](https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-behaviour), the authors explain this:
> The default behaviour of the action is to create a pull request that will be continually updated with new changes until it is merged or closed. Changes are committed and pushed to a fixed-name branch, the name of which can be configured with the branch input. Any subsequent changes will be committed to the same branch and reflected in the open pull request.

More details in the README itself.

We can change this cadence to whatever, or even have these PRs auto-merged (like some Dependabot PRs we have) or commit directly to `main` (IMO worse).

Reply:
yea, we might loop out of free github action runtime.. Sooner or later? Not sure of long a single run will take. We "only" have 2000 minutes per month. And we also want to keep some minutes for other PRs of course.

| Plan	         |      Storage  |	Minutes (per month) |
|------------------|------------------|------------------------------------|
|GitHub Free |	   500 MB |	2,000 |


Source: https://docs.github.com/en/billing/managing-billing-for-your-products/managing-billing-for-github-actions/about-billing-for-github-actions